tokio-trace/test_static_max_level_features/Cargo.toml

```diff
@@ -0,0 +1,12 @@
+[package]
+name = "test_cargo_max_level_features"
+version = "0.1.0"
+publish = false
+
+[[bin]]
```

Comment:
Since the test was moved into `tests`, the whole `[[bin]]` section needs to be removed.

Doing so will fix the CI build.